Packages/com.unity.inputsystem/InputSystem/Editor/ControlPicker/InputControlPathEditor.cs

```diff
@@ -206,12 +210,16 @@ private void SetExpectedControlLayoutFromAttribute(SerializedProperty property)
         private GUIContent m_PathLabel;
         private string m_ExpectedControlLayout;
         private string[] m_ControlPathsToMatch;
-        private InputControlScheme[] m_ControlSchemes;
-        private bool m_NeedToClearProgressBar;
 
         private InputControlPickerDropdown m_PickerDropdown;
         private readonly InputControlPickerState m_PickerState;
-        private InputActionRebindingExtensions.RebindingOperation m_RebindingOperation;
+
+        /// <summary>
+        /// This property is only set from this class in order to communicate that we're showing the dropdown at the moment
+        /// It's employed to skip auto-saving, because that complicates updating the internal SerializedProperties.
+        /// Unfortunately, we can't use IMGUIDropdownVisible from the setings provider because of the early-out logic in there.
+        /// </summary>
+        public static bool IsShowingDropdown { get; private set; }
```

Comment:
Yes, that is a static property. However, it's only set from one place (private set), and it won't leak memory. Also, we somehow happen to have two drop-down editor windows at the same time (which QA will correct me is barely possible). Even if somehow somehow opens two drop-down thingies at the same time, I believe we're fine here, as the more drop-down windows we have, the less sense it makes to auto-save during those sensitive moments of time. Hope that makes sense.